src/App.js

```diff
@@ -84,6 +87,44 @@ const App = ({ setClient }) => {
   useInitI18n(client)
   useInitBackup(client)
   useOsReceiveApi()
+  const { restartApp } = useRestartContext()
+
+  useEffect(() => {
+    const doAsync = async () => {
+      if (!client) return
+
+      const {
+        data: {
+          attributes: { colorScheme }
+        }
+      } = await client.fetchQueryAndGetFromState({
+        definition: Q('io.cozy.settings').getById('io.cozy.settings.instance'),
+        options: {
+          as: 'io.cozy.settings/io.cozy.settings.instance',
+          singleDocData: true
+        }
+      })
+
+      setStoredColorScheme(colorScheme)
+
+      await client.plugins.realtime.subscribe(
+        'updated',
+        'io.cozy.settings',
+        updatedSettings => {
+          // We can updated the stored color scheme but in this case
+          // - webview will be updated because of its internal cozy-realtime usage
+          // - statusbar and navbar won't be okay because setFlagshipUI is not called again
+          // - next transtion from a cozy app to another will be okay
+          setStoredColorScheme(updatedSettings.colorScheme)
```

Comment:
Any idea on how to update the current statusbar and navbar (see video) ?

https://github.com/cozy/cozy-flagship-app/assets/10849491/782f1804-1b10-4ed2-afcb-4582879472d5